source/lambda/apiprocessor/redaction.py

```diff
@@ -33,7 +32,7 @@
 MAXIMUM_NUM_OF_HEADERS_FOOTERS = 3
 
 # we expect a redacted item to have 6 keys: page, top, left, width, height label
-REDACTED_ITEM_KEYS = 6
+REDACTED_ITEM_NUM_KEYS = 5
```

Comment:
Can we have REQUIRED_REDACTED_ITEM_NUM_KEYS=5 & OPTIONAL_REDACTED_ITEM_NUM_KEYS=1?